db/migrations/postgres/000097_posts_originalid_index.up.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS idx_posts_original_id ON Posts(originalid);
```

Comment:
I see that you added a commit changing this to lowercase. Is that needed?

Reply:
Yes, probably not necessary but when I check similar previous PRs (https://github.com/mattermost/mattermost-server/pull/19841 and https://github.com/mattermost/mattermost-server/pull/19799), they used lowercase so I wanted to be consistent